opensrp-chw-hf/src/main/java/org/smartregister/chw/hf/HealthFacilityApplication.java

```diff
@@ -227,8 +226,6 @@ public void onCreate() {
         }
         // set up processor
         FamilyLibrary.getInstance().setClientProcessorForJava(HfClientProcessor.getInstance(getApplicationContext()));
-
-        Mapbox.getInstance(this, BuildConfig.MAPBOX_SDK_ACCESS_TOKEN);
```

Comment:
I think we can leave this here

Reply:
I've moved to the App level but PR failed with 
```java
org.smartregister.chw.hf.activity.LoginActivityTest > testUserNameEditTextIsInitialized FAILED
    java.lang.ExceptionInInitializerError at ConnectivityReceiver.java:43
        Caused by: java.lang.NullPointerException at File.java:279
```

moving back to the `FamilyRegisterActivity.java`

Reply:
It's recommended to have it in the Application class instead. I don't think the test failing is a reason to remove it unless the tested functionality on the device is the issue since the tests will still fail. The tests are passing because there is no `FamilyRegisterActivityTest`

Reply:
I'll still need this in the Application class before approving this PR

Reply:
done